src/ordered_robin_dict.jl

```diff
@@ -0,0 +1,309 @@
+using DataStructures: RobinDict;
```

Comment:
But DataStructures has a dependency on OrderedCollections
so you can’t have OrderedCollections depend on DataStructures

The OrderedRobinDict has a RobinDict typed field